site/ko/guide/eager.ipynb

```diff
@@ -0,0 +1,1291 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vbFnP-yLqEgx"
+      },
+      "source": [
+        "모델을 저장하거나 읽어들이기 위해서, `tf.train.Checkpoint`는 숨겨진 변수를 요구하지 않고 객체 내부 상태를 저장합니다. \n",
+        "`옵티마이저`와 `모델`, 전역 단계 상태를 기록하려면 `tf.train.Checkpoint`에 전달하면 됩니다:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "attributes": {
+          "classes": [
+            "py"
+          ],
+          "id": ""
+        },
+        "colab": {},
+        "colab_type": "code",
+        "id": "hWZHyAXMqEg0"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "\n",
+        "model = tf.keras.Sequential([\n",
+        "  tf.keras.layers.Conv2D(16,[3,3], activation='relu'),\n",
+        "  tf.keras.layers.GlobalAveragePooling2D(),\n",
+        "  tf.keras.layers.Dense(10)\n",
+        "])\n",
+        "optimizer = tf.keras.optimizers.Adam(learning_rate=0.001)\n",
+        "checkpoint_dir = 'path/to/model_dir'\n",
+        "if not os.path.exists(checkpoint_dir):\n",
+        "  os.makedirs(checkpoint_dir)\n",
+        "checkpoint_prefix = os.path.join(checkpoint_dir, \"ckpt\")\n",
+        "root = tf.train.Checkpoint(optimizer=optimizer,\n",
+        "                           model=model)\n",
+        "\n",
+        "root.save(checkpoint_prefix)\n",
+        "root.restore(tf.train.latest_checkpoint(checkpoint_dir))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "R-ITwkBCF6GJ"
+      },
+      "source": [
+        "Note: 대부분의 반복 훈련 과정에서 변수는 `tf.train.Checkpoint.restore`가 호출된 이후에 생성됩니다.\n",
+        "이러한 변수는 생성되자마자 복원될 것이므로 단언문을 통해 체크포인트가 완벽히 적재되었다는 것을 보장받을 수 있습니다.\n",
+        "자세한 내용은 [훈련 체크포인트 가이드](./checkpoint.ipynb)를 참고하세요."      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "3yoD0VJ7qEg3"
+      },
+      "source": [
+        "### 객체 지향형 지표\n",
+        "\n",
+        "`tf.keras.metrics`는 객체로 저장됩니다.\n",
+        "새로운 데이터를 이 객체에 전달하여 지표를 수정하고 `tf.keras.metrics.result` 메서드를 사용해 그 결과를 얻습니다.\n",
+        "예를 들어:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "attributes": {
+          "classes": [
+            "py"
+          ],
+          "id": ""
+        },
+        "colab": {},
+        "colab_type": "code",
+        "id": "9ccu0iAaqEg5"
+      },
+      "outputs": [],
+      "source": [
+        "m = tf.keras.metrics.Mean(\"loss\")\n",
+        "m(0)\n",
+        "m(5)\n",
+        "m.result()  # =\u003e 2.5\n",
+        "m([8, 9])\n",
+        "m.result()  # =\u003e 5.5"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "aB8qWtT955pI"
+      },
+      "source": [
+        "### 요약과 텐서보드\n",
```

Comment:
`summary` in TF is something like logging. We don't translate it as '요약' to prevent readers from misunderstanding.
Please change '요약과' --> '서머리(summary)와'.

Reply:
Thanks, fixed